resolwe/test_helpers/test_runner.py

```diff
@@ -48,6 +308,45 @@ def add_arguments(cls, parser):
             help="File which describes what kind of changes are available"
         )
 
+        parser.add_argument(
+            '--keep-data', dest='keep_data', action='store_true',
+            help="Prevent test cases from cleaning up after execution"
+        )
+
+        parser.add_argument(
+            '--no-mock-purge', dest='no_mock_purge', action='store_true',
+            help="Do not mock purging functions (implies --keep-data)"
+        )
+
+    def run_suite(self, suite, **kwargs):
+        """Run the test suite with manager workers in the background."""
+        # due to the way the app modules are imported, there's no way to statically override settings
```

Comment:
Wrap this comment at 80 or 99 characters (your choice) and convert it to sentences.